### ISSUE TITLE
As a user, there should be a clear place to enter new developments on a case

### ISSUE THREAD

**Issue by trystant** [EBWiki/EBWiki#107]
So, I saw this development on the [Eric Garner](http://ebwiki.org/articles/eric-garner) case, where the medical examiner was an ex NYPD officer. I wanted to add it to the case as exists. Currently, I have two options:
1. I can add it as a link that shows up in "Additional Resources", but how can we show that it's something that was added relatively recently? Right now, the links are being listed in chronological order; I think that reversing that order and adding information on when the link was added to the case would help.
1. I can make a comment about it, but there are several issues with the comments as is. I am going to enter a separate issue about comments and refer to it later, but for now here is a screenshot with comments in its current form ![screenshot](http://grandkru.com/images/eric_garner_comment.png)):

**Comment by campgurus** [comment]
This is a fundamental issue that we need to figure out, relating to updates, possibly timelines, how we display the most recent additions to a case, etc.  I think this merits some discussion and a decision, before we can implement a solution.

**Comment by campgurus** [comment]
Just reread this. Adding a date to the links is easy to do. here is what it looks like:

<img width="871" alt="screen shot 2016-05-09 at 6 14 55 am" src="https://cloud.githubusercontent.com/assets/1260659/15110188/76218b1e-15ad-11e6-8d9f-7140ea21e26e.png">

I think it is helpful and I will push to master, but we still have work to do on the fundamental issue you have raised, so this issue will remain open.

**Comment by campgurus** [comment]
Fixing the history tab to be more readable as documented by #71 will help with this, but it is not a perfect solution. How does Wikipedia deal with this question?

**Comment by trystant** [comment]
I don't know that Wikipedia automatically highlights new developments in a
case.

On Mon, May 9, 2016 at 6:22 AM, campgurus <notifications@github.com> wrote:

> Fixing the history tab to be more readable as documented by #71
> <https://github.com/BOWiki/BOW/issues/71> will help with this, but it is
> not a perfect solution. How does Wikipedia deal with this question?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/BOWiki/BOW/issues/107#issuecomment-217828526>
>



-- 

Mark Nyonmark@grandkru.com

blog.grandkru.com

github.com/trystant

**Comment by rlgreen91** [comment]
Here's my idea: I think we should follow a similar interface to Wikipedia, where we have an edit link in the header and for each section after the name (ideally to a form for that section).  That allows us to remove the edit button from the right side, where it's mostly hidden.

I'm actually not a big fan of having the dates after the added links.  I don't see that it really provides me with any information that I would be interested in knowing.  To me, the "last updated" line should include any updates, including linked content, and those dates should go away.  That means that change would have to wait until we get rid of the links model.  We can then use the history page to give a more granular look into what exactly was changed and when.

**Comment by trystant** [comment]
We currently have dates for the link section as well as a header for each section. I believe that it's clearer where to place information.
